### DISCUSSION TITLE
Seed Freezing!

### DISCUSSION THREAD

**Discussion by puchungotv** [tearust/teaproject#13] [Q&A] [answered]
What is the Purpose of Freezing Seeds?

**Comment by kevingzhang** [comment, marked as answer]
There are many:
1. Release seeds gradually. We do not want early investors to put all CML seeds to mining at the first block. 
2. Keep a steady mining power increase over time. If all seeds join mining at the same time, they might die at the same time too. We do not want the mining nodes increase or decrease sharply
3. You can sell / trade / collateral your frozen seeds. That is your future assets, use the Defi tools to maximize your revenue.
